tests/integration_tests/async_events/api_tests.py

```diff
@@ -30,6 +31,7 @@
 from tests.integration_tests.test_app import app
 
 
+@pytest.skip(reason="Needs to investigate this test", allow_module_level=True)
```

Comment:
This one does not seem related to the legacy charts removal. Need to investigate in more detail.

Reply:
No. I will investigate when re-enabling the test in a follow-up.

Reply:
gotcha. thank you!

Reply:
@Vitor-Avila I tried to restore this test but it's throwing an un-related error. If you want to take a stab at it.. 🤷🏼‍♂️

Reply:
@michael-s-molina I tested locally and it seems the tests work if `GLOBAL_ASYNC_QUERIES` FF is set to `True`, which I think makes sense? I'll double check that and if so push a PR.

Reply:
@michael-s-molina pushed the change here: https://github.com/apache/superset/pull/32062 it worked locally -- hopefully it works through CI as well 🙏

Reply:
ooof, failed in the CI 😢

Reply:
Yeah, I faced the same error trying to make it work 😢

Reply:
ok @michael-s-molina got it to work 🙌 could you please review? https://github.com/apache/superset/pull/32062